### ISSUE TITLE
Any possibility to have a body text version of Cormorant/Cormorant Garamond for ereaders

### ISSUE THREAD

**Issue by Vakke** [CatharsisFonts/Cormorant#71]
Hello,

I read an older "issue" where person asked for more font weights and there you had said that the font is not meant to be a body text, rather a font for larger screens and sizes.

However! I would love to have this font as a body text variant.

I've tested EB Garamond and other Adobe/ITC variants of the Garamond and I like yours the best. Only problem for ereading is that the font is really even at medium and some of the lines extremely so, for example g :s lower part. 

So this is not an "issue" but rather a question and a plead for a more body text oriented Cormorant/Cormorant Garamond font variant. Something to be enjoyed for example in ereaders!

**Comment by CatharsisFonts** [comment]
Hi Vakke, 

I don't have the capacity for this sort of work in the foreseeable future.

Have you tried Ysabeau? I use it for my Kindle and I love it. 🤓 

Cheers, Christian

**Comment by Vakke** [comment]
I totally understand this in great pain! 😄 

I've actually tried it and many other sans serifs but I prefer the old style serif fonts and how the letters just dance in vision! Well if ever get the inspiration and time to mold the font for a body text style to ereaders I'm all in to help on the testing if that is needed.

Thank you for your great work and I'm still going to enjoy Cormorant Garamond on my Kobo even though it's not designed for it!

**Comment by cmahte** [comment]
I think I might have been the earlier request. 

and while I'm still waiting for CT to loop back around to it.... 

Here are some "almost" fonts that you should consider: 

[Coelacanth from Font Library](https://fontlibrary.org/en/font/coelacanth)

- Note: I have this font locally broken into 6 different folders: Coelacanth, ..Caption, ..Display, ..Pearl, ..SubCap, ..SubHd. I mention this specifically that I'm pointing you to the base flavor, but you should look at the pearl faces separately, especially for e-readers. I found it a bit too 'edgy' to use... that is the edges at large sizes are too straight and distracting.  But for a book font, it's a bit more interesting than EBGaramond. 

Linux Libertine (which comes with LibreOffice... never chased it back to a source.)

- If I had evaluated this font before EBGaramond, I'd probably be using it instead.  It's tall, and it's not a Garamondesque, yet it is compatible. 

[Alegreya from Huerta Typographica](https://www.huertatipografica.com/en/fonts/alegreya-ht-pro)

This font is not like Garamond, but it is tall and readable. The only issue I had with it  is I need readable down to 6.5 points, and the vertical strokes decrease in thickness from top to bottom. At the minimum size it was noticably stressful. but if you're needs are more in the 8-10 point range, it's a fully developed look, with a matching sans font.

**Comment by Vakke** [comment]
> 

Yes it was exactly you and SunnyKodukula on the Font weight thread!

Thank you for these suggestions, I'm familiar with the Linux Libertine and I've actually enjoyed it quite a bit. I found out a bit ago that there is a font called [Libertinus](https://github.com/alerque/libertinus) which a fork of Libertine as it hasn't been updated for over ten years and should've fixed some problems the original had. 

But Coelacanth and Alegreay are new to me, I'll download both and give them a change! They looks great but damn I just like the Garamond.

May I ask that do you use the EB Garamong on your reading and if you do what is your prefered device to use it with? It seems that the font is a bit different in different ereaders and different filetypes used. I found the lower case f and g to be too thin in my Kobo and have thought to add a bit weight to them in FontForge when I have spare time to use.

**Comment by RGB-es** [comment]
@cmahte @Vakke Be aware that the Font Library has really outdated versions of all its fonts (except for a couple of project, the whole site seems unmaintained). You can find the original Coelacanth repo here on github

https://github.com/Fuzzypeg/Coelacanth 

Under Releases → Tags where is version 0.008

**Comment by Gl17chV** [comment]
Another option in the same vein is Jannon, which has historically been overshadowed by its similarity to [Garamond](https://fontsinuse.com/uses/14168/the-garamond-types-considered-in-the-fleuron-).

I'm currently captivated by François Rappo's version, [JJannon](https://optimo.ch/typefaces/jjannon). I use it as body text, and it's perfect; the numeral style is modern, a _Book_ version and the _Display_ version has a feel very similar to Cormorant G. Sadly, it only supports _Latin_. [Jannon Neo](https://www.stormtype.com/families/jannon), which supports Greek and Cyrillic, exists; inconveniently, it has a much more modern style, leaning more towards the Bodoni style. [Jannon Pro](https://www.myfonts.com/collections/jannon-pro-font-storm?srsltid=AfmBOor_XJ-XeZG_NwqtzrLt37A4Z951O7upaZit5zx2aCMKIxS0aUqR), by the same author, retains the classic style. I haven't purchased it because it seems the _Antique_ version doesn't support Greek. All of these are paid fonts, and I don't know if a free version of Jannon exists.
